packages/consensus/fcob/consensusmechanism.go

```diff
@@ -133,13 +150,42 @@ func (f *ConsensusMechanism) ProcessVote(ev *vote.OpinionEvent) {
 				f.onPayloadOpinionFormed(messageID, opinion.liked)
 			}
 		})
+	case vote.TimestampType:
+		messageID, err := tangle.NewMessageID(ev.ID)
+		if err != nil {
+			f.Events.Error.Trigger(err)
+			return
+		}
+
+		f.Storage.TimestampOpinion(messageID).Consume(func(opinion *TimestampOpinion) {
+			opinion.SetLiked(ev.Opinion)
+			opinion.SetLevelOfKnowledge(Two)
+		})
+
+		// eligible when timestamp is ok but confirmed when timestamp and tx is correct
+		f.setEligibility(messageID)
+
+		f.setTimestampOpinionDone(messageID)
+
+		if f.messageDone(messageID) {
```

Comment:
Then this part will not be executed. It will be executed instead when e.g. voting about a payload finishes. Or in case of a correct message, it will be marked as confirmed either in timestamp or payload evaluation, depending on which one will be executed second.

Reply:
ok thanks. I think this would be good information to have here. could you add this.